lib/Dialect/TritonGPU/IR/Dialect.cpp

```diff
@@ -257,8 +264,8 @@ SmallVector<unsigned> getOrderForDotOperand(unsigned opIdx, unsigned rank) {
   //
   // The relation between contiguity and order is only relevant if the layout
   // interfaces with HBM, as is the case when we load tensor from HBM to
-  // registers in the dot layout to bypass LDS. When bypassing LDS, we make the
-  // following assumptions about tensor layouts:
+  // registers in the dot layout to bypass LDS. When bypassing LDS, we make
+  // the following assumptions about tensor layouts:
   // - Tensor A (opIdx == 0) is considered to be row-major.
   // - Tensor B (opIdx == 1) is considered to be column-major.
   //
```

Comment:
The `swap` below should be between `[rank -  2]` and `[rank - 1]`.

Reply:
I think it's fine because it's order[0] and order[1]

Reply:
You are right. It's the comment below that's incorrect. It should be:
```
  // - For opIdx == 0, we assume an order of [2, 1, 0] for 3D tensors.
  // - For opIdx == 1, we assume an order of [1, 2, 0] for 3D tensors.
```